RecoTracker/LSTCore/src/alpaka/Segment.h

```diff
@@ -638,20 +638,20 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
       }
       alpaka::syncBlockThreads(acc);
 
-      // Occupancy matrix for 0.8 GeV pT Cut
+      // Occupancy matrix for 0.8 GeV pT Cut, 99.99%
       constexpr int p08_occupancy_matrix[4][4] = {
-          {572, 300, 183, 62},  // category 0
-          {191, 128, 0, 0},     // category 1
-          {0, 107, 102, 0},     // category 2
-          {0, 64, 79, 85}       // category 3
+          {740, 314, 230, 60},  // category 0
+          {1097, 693, 0, 0},    // category 1
```

Comment:
a factor of 5 increase here: was the target quantile incidentally lower before?

Reply:
Yes, category 1 here was set to 99% before. I increased it to 99.99% to match the other categories.